mitmproxy/command.py

```diff
@@ -271,7 +278,7 @@ def command(path):
     def decorator(function):
         @functools.wraps(function)
         def wrapper(*args, **kwargs):
-            verify_arg_signature(function, args, kwargs)
+            verify_arg_signature(function, args, tuple(kwargs.items()))
```

Comment:
Doesn't look like this is working, as some of the commands' arguments themselves are lists. Looks like they need to be flattened for `@lru_cache` to accept them. However, the values themselves aren't necessary, are they? Any way we can pass just the types?

Reply:
This is really surprising as *args is consistently a tuple on my system. 🤷‍♂️ 
Anyways, fix attempt below.

Reply:
I admit I didn't look into it too closely, but I think some of the arguments may contain objects that in turn contain non-hashable types, like lists?